test/stripe/util_test.rb

```diff
@@ -52,10 +52,10 @@ class UtilTest < Test::Unit::TestCase
         # *The key here is the order*. In order to be properly interpreted as
         # an array of hashes on the server, everything from a single hash must
         # come in at once. A duplicate key in an array triggers a new element.
-        ["f[][bar]", "1"],
-        ["f[][foo]", "2"],
-        ["f[][baz]", "3"],
-        ["f[][foo]", "4"],
+        ["f[][foo]", "1"],
+        ["f[][ghi]", "2"],
+        ["f[][foo]", "3"],
+        ["f[][bar]", "4"],
```

Comment:
Note that I've changed the test here because although the comment above it is accurate, the test itself was actually wrong. The server would have interpreted this as:

``` ruby
[
  { :bar => "1", :baz => "3" :foo => "2" },
  { :foo => "4" },
]
```

Which is obviously incorrect.